docs/marketplace/dev/technical-enhancement/migration-guide-upgrade-to-angular-v15.md

```diff
@@ -0,0 +1,457 @@
+| UserMerchantPortalGui                       | >= 2.0.0  |
+| ZedUi                                       | >= 2.0.0  |
+
+If not, update module versions manually or by using the following command:
+
+```bash
+composer update spryker/dashboard-merchant-portal-gui spryker/gui-table spryker/merchant-profile-merchant-portal-gui spryker/product-merchant-portal-gui spryker/product-offer-merchant-portal-gui spryker/sales-merchant-portal-gui spryker/security-merchant-portal-gui spryker/user-merchant-portal-gui spryker/zed-ui
+```
+
+2. Regenerate the data transfer object:
+
+```bash
+console transfer:generate
+```
+
+## 2) Update Webpack
+
+Before starting the migration, make sure that Webpack is v5; if it's v4, the [Webpack migration guide](/docs/scos/dev/front-end-development/migration-guide-upgrade-to-webpack-v5.html) is required.
+
+## 3) Update NPM dependencies
+
+{% info_block infoBox "Info" %}
+
+Make sure you are using [Node 16 or later](https://nodejs.org/dist/latest-v16.x/).
+
+{% endinfo_block %}
+
+1. In `package.json`, update or add the following dependencies:
+
+```json
+{
+    "dependencies": {
+        "@angular/animations": "~15.0.3",
+        "@angular/cdk": "~15.0.3",
+        "@angular/elements": "~15.0.3",
+        "@angular/forms": "~15.0.3",
+        "@angular/router": "~15.0.3",
+        "@angular/common": "~15.0.3",
+        "@angular/compiler": "~15.0.3",
+        "@angular/core": "~15.0.3",
+        "@angular/platform-browser": "~15.0.3",
+        "@angular/platform-browser-dynamic": "~15.0.3",
+        "rxjs": "~7.5.7",
+        "zone.js": "~0.12.0"
+    },
+    "devDependencies": {
+        "@angular-builders/custom-webpack": "~15.0.0",
+        "@angular-devkit/build-angular": "~15.0.3",
+        "@angular-eslint/builder": "~15.0.0",
+        "@angular-eslint/eslint-plugin": "~15.0.0",
+        "@angular-eslint/eslint-plugin-template": "~15.0.0",
+        "@angular-eslint/schematics": "~15.0.0",
+        "@angular-eslint/template-parser": "~15.0.0",
+        "@angular/cli": "~15.0.3",
+        "@angular/compiler-cli": "~15.0.3",
+        "@angular/language-service": "~15.0.3",
+        "@nrwl/cli": "~15.0.7",
+        "@nrwl/jest": "~15.0.7",
+        "@nrwl/workspace": "~15.0.7",
+        "@types/jest": "~28.1.1",
+        "@typescript-eslint/eslint-plugin": "~5.44.0",
+        "@typescript-eslint/parser": "~5.44.0",
+        "eslint": "~8.28.0",
+        "eslint-plugin-deprecation": "~1.3.3",
+        "jest": "~28.1.3",
+        "jest-environment-jsdom": "~28.1.1",
+        "jest-preset-angular": "~12.2.3",
+        "nx": "~15.0.7",
+        "ts-jest": "~28.0.8",
+        "ts-node": "~10.9.1",
+        "typescript": "~4.8.4"
+    }
+}
+```
+
+2. Remove the following dependencies:
+
+```json
+{
+    "devDependencies": {
+        "@nrwl/tao": "~12.10.1",
+        "codelyzer": "~6.0.0",
+        "typescript-eslint-parser": "~22.0.0"
+    }
+}
+```
+
+3. Update and install package dependencies:
```

Comment:
@VadymSachenko They shouldn't be as separate items, because the changes must be in the `package.json`.
So it should be mentioned in each item or be a part of the one item as before.